pyciemss/ouu/ouu.py

```diff
@@ -128,11 +126,16 @@ def propagate_uncertainty(self, x):
             with torch.no_grad():
                 x = np.atleast_1d(x)
                 # Combine existing interventions with intervention being optimized
+                intervention_list = [
+                    deepcopy(self.fixed_static_parameter_interventions)
+                ]
+                intervention_list.extend(
+                    [self.interventions(torch.from_numpy(x))]
+                    # if not isinstance(self.interventions(torch.from_numpy(x)), list)
```

Comment:
Could you remove this commented out code?

Reply:
Done